quiz.js

```diff
@@ -5,54 +5,84 @@
 var heightField = document.getElementById("height");
 var charField = document.getElementById("character");
 var submitBTN = document.getElementById("growButton");
-var logPlace;
+var printedTree = "";
 
-var tree = {
+
+
+// makes the user input for height and character update tree keys and building the object
+
+
+function getInput(){
+	// console.log("getInput is running")
+	var tree = {
 	height: 0,
 	character: 0
 }
+	tree.height = heightField.value.substring(0,2);//limits the characters read
+	tree.character = charField.value.charAt(0);
+	buildTree(tree);
+
+};
 
```

Comment:
What's this empty space for? Try to create consistent spacing in your code. It makes it much easier to read.